builtin/providers/openstack/data_source_openstack_networking_network_v2.go

```diff
@@ -0,0 +1,102 @@
+package openstack
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/networks"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/subnets"
+)
+
+func dataSourceNetworkingNetworkV2() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNetworkingNetworkV2Read,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"matching_subnet_cidr": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"region": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				DefaultFunc: schema.EnvDefaultFunc("OS_REGION_NAME", ""),
+			},
+			"admin_state_up": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"shared": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
Can you move `admin_state_up` and `shared` to the bottom of the list like [here](https://github.com/hashicorp/terraform/blob/master/builtin/providers/openstack/data_source_openstack_images_image.go#L84)?